sig-federation/CONTRIBUTING.md

```diff
@@ -0,0 +1,294 @@
+# Contributing
+
+The process for contributing code to Kubernetes via the sig-multicluster [community][community page].
+
+## TL;DR
+
+- The sig-multicluster [community page] lists sig-multicluster [leads]
+  and group [meeting] times.
+- Request a feature by making an [issue] and mentioning
+  `@kubernetes/sig-multicluster-feature-requests`.
+- Write a design proposal before starting work on a new feature.
+- Write [tests]!
+
+## Before You Begin
+
+Welcome to the Kubernetes sig-multicluster contributing guide.  We are excited
+about the prospect of you joining our [community][community page]!
+
+Please understand that all contributions to Kubernetes require time
+and commitment from the project maintainers to review the ux, software
+design, and code.  Mentoring and on-boarding new contributors is done
+in addition to many other responsibilities.
+
+### Understand the big picture
+
+- Complete the [Kubernetes Basics Tutorial].
+
+### Agree to contribution rules
+
+Follow the [CLA signup instructions](../CLA.md).
+
+### Adopt an issue
+
+New contributors can try the following to work on an existing bug or approved design:
+
+- In [slack][slack-messages] (signup [here][slack-signup]),
+  @mention a [lead][leads] and ask if there are any issues you could pick up.
+  We also maintain a list of [multi cluster issues where help is wanted][multicluster_help_wanted_issues].
+  Most of them are not very complex, so that's probably a good starting point.
+  Leads can recommend issues that have enough priority to receive PR review bandwidth.
+- Send an email to the _kubernetes-sig-multicluster@googlegroups.com_ [group]
+
+  > Subject: New sig-multicluster contributor _${yourName}_
+  >
+  > Body: Hello, my name is _${yourName}_.  I would like to get involved in
+  > contributing to the Kubernetes project.  I have read all of the
+  > user documentation listed on the community contributing page.
+  > What should I do next to get started?
+
+- Attend a sig-multicluster [meeting] and introduce yourself as looking to get started.
+
+### Bug lifecycle
+
+#### Filing a bug
+
+1. An [issue] is filed that
```

Comment:
Is there a way to see how buildcop rotation works, what they will view as higher priority and how to poke the one on duty if it falls into that criteria? These instructions are great but how does someone know after filing a bug what the expectations are to have it looked at?

Reply:
Am adding ONCALL information in https://github.com/kubernetes/community/pull/1145.
Will link to it here once that PR merges.